.env.dev

```diff
@@ -9,5 +9,5 @@ CUSTOMS_SERVER_URL=none
 RESEND_BLACKOUT_PERIOD=0
 LOCKOUT_ENABLED=true
 SNS_TOPIC_ARN=disabled
-TRUSTED_JKUS=http://127.0.0.1:8080/.well-known/public-keys,http://127.0.0.1:10139/.well-known/public-keys
+TRUSTED_JKUS=http://127.0.0.1:8080/.well-known/public-keys,http://127.0.0.1:10139/.well-known/public-keys,http://127.0.0.1:9000/.well-known/public-keys
```

Comment:
This was brought over from original PR, but I think it can be removed. It was added so that the `mail-dev` tool could intercept emails for visual inspection.